src/hevm/src/EVM/TTY.hs

```diff
@@ -725,21 +719,21 @@ drawVmBrowser ui =
               (\selected (k, c') ->
                  withHighlight selected . txt . mconcat $
                    [ fromMaybe "<unknown contract>" . flip preview dapp' $
-                       ( dappSolcByHash . ix (view codehash c')
+                       ( dappSolcByHash . ix (maybeHash c)
```

Comment:
Is `c` or `c'` the right contract to use here? Since this is inside the closure folding over `c'`s I would suspect `c'`

Reply:
nice catch :)